[2022-08-24 01:54:31,853] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.insere_dados_na_tabela manual__2022-08-24T01:54:26.846384+00:00 [queued]>
[2022-08-24 01:54:31,868] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.insere_dados_na_tabela manual__2022-08-24T01:54:26.846384+00:00 [queued]>
[2022-08-24 01:54:31,869] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-24 01:54:31,870] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-24 01:54:31,870] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-24 01:54:31,891] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): insere_dados_na_tabela> on 2022-08-24 01:54:26.846384+00:00
[2022-08-24 01:54:31,899] {standard_task_runner.py:52} INFO - Started process 86 to run task
[2022-08-24 01:54:31,905] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'insere_dados_na_tabela', 'manual__2022-08-24T01:54:26.846384+00:00', '--job-id', '9636', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmp6xxkrvtg', '--error-file', '/tmp/tmpp24hx3dv']
[2022-08-24 01:54:31,907] {standard_task_runner.py:80} INFO - Job 9636: Subtask insere_dados_na_tabela
[2022-08-24 01:54:32,008] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.insere_dados_na_tabela manual__2022-08-24T01:54:26.846384+00:00 [running]> on host b8d797566373
[2022-08-24 01:54:32,118] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=insere_dados_na_tabela
AIRFLOW_CTX_EXECUTION_DATE=2022-08-24T01:54:26.846384+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-24T01:54:26.846384+00:00
[2022-08-24 01:54:32,194] {logging_mixin.py:115} INFO - TABELA CRIADA
[2022-08-24 01:54:32,195] {python.py:173} INFO - Done. Returned value was: None
[2022-08-24 01:54:32,214] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=insere_dados_na_tabela, execution_date=20220824T015426, start_date=20220824T015431, end_date=20220824T015432
[2022-08-24 01:54:32,276] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-24 01:54:32,341] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
